docs/user-guide/styling-your-docs.md

```diff
@@ -77,9 +88,11 @@ h1 {
 
         extra_css: [extra.css]
 
-After making these changes, they should be visible when you run
-`mkdocs serve` - if you already had this running, you should see that the CSS
-changes were automatically picked up and the documentation will be updated.
+After addding these files and making the changes to `mkdocs.yml`,
```

Comment:
This change is completely unnecessary. Part of the "changes" includes adding the files. Why do we need to mention that specifically?